sal-service/src/main/java/org/ow2/proactive/sal/service/service/TaskBuilder.java

```diff
@@ -60,10 +60,14 @@ public class TaskBuilder {
     private ScriptTask createEmsDeploymentTask(EmsDeploymentRequest emsDeploymentRequest, String taskNameSuffix,
             String nodeToken) {
         LOGGER.debug("Preparing EMS deployment task");
+        String preScriptFileName = "emsdeploy_prescript.sh";
```

Comment:
I would have moved script name variables into dedicated final static variables. But it is a matter  of taste!

Reply:
Good point I will merge this PR and change all the names to static variables in a new PR